tests/test_utils.py

```diff
@@ -96,3 +96,20 @@ def test_process_query(input, output):
 def test_process_query_bytes():
     with raises(ValueError):
         _process_query({"url": b"https://example.com"})
+
+
+@pytest.mark.parametrize(
+    "custom_user_agent,expected",
+    (
+        (
+            None,
+            f'python-zyte-api/{__version__}',
+        ),
+        (
+            'scrapy-zyte-api/0.11.1',
+            f'python-zyte-api/{__version__}, scrapy-zyte-api/0.11.1',
```

Comment:
@kmike What do you think about using a space as separator instead, in line with https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax ?

```suggestion
            f'python-zyte-api/{__version__} scrapy-zyte-api/0.11.1',
```

Reply:
@kmike @Gallaecio also, could we agree on order of packages?

Reply:
Using spaces sounds right. +0.5 to put scrapy-zyte-api as a first entity in scrapy-zyte-api's user agent.